{
	"id": "CVE-2020-13281",
	"description": "For GitLab before 13.0.12, 13.1.6, 13.2.3 a denial of service exists in the project import feature",
	"scope": "local",
	"packages": [
		{
			"name": "gitlab",
			"status": "resolved",
			"urgency": "not yet assigned",
			"fixed_version": "13.2.3-2",
			"repository": [
				{
					"name": "sid",
					"version": "16.6.5-10"
				}
			]
		}
	]
}
